src/Prelude.hs

```diff
@@ -42,3 +45,12 @@ import qualified Colog (Message, WithLog)
 
 -- | 'Colog.WithLog' alias specialized to 'Message' data type.
 type WithLog env m = Colog.WithLog env Colog.Message m
+
+{- | This function takes a delimeter and a delimeter seperated value,
+and returns a list of @Text@ values stripped of excess whitespace.
+Note that it returns an empty list when an empty delimeter seperated value is
+passed in. This prevents the value @[""]@ from being returned.
+-}
+splitAndStrip :: Text -> Text -> [Text]
+splitAndStrip _ ""       = []
```

Comment:
This function is not used in this PR. But it's used in another PR with `MonadCabal` (if I'm not wrong). Need to remember to not have duplication after resolving conflicts.

Reply:
That’s correct.

Reply:
@chshersh I fixed this.